packages/core/src/delegatedTaggedState.ts

```diff
@@ -0,0 +1,6 @@
+import { StateVariant } from "./taggedState";
+
+export type DelegatedFlatStateVariant<
```

Comment:
Let's move the implementation into the `taggedState.ts`

Reply:
I guess the `delegatedTaggedState.spec.ts` I can move into the `taggedState.spec.ts` too

Reply:
https://github.com/MOZGIII/tagged-state/pull/1/commits/d8154e1df95ee0cee6d3be9dc4d32727a4c870e5